new_docs/DOCSTRUCTURE.md

```diff
@@ -0,0 +1,53 @@
+# doc-structure
+this is a repository for the new documentation structure
+
+
+-----------------------------------------------------------------------------
+
+
+Chapter 1 Concepts of thin-edge.io 
+
+(In this chapter terminology and concepts of thin-edge.io should be explained. Not only text should be used but also (simple/schematic) images. See the cumulocity concepts guide for examples. What we have now is too developer oriented, concepts should be more general described
+
+    1.0 Overview - Generic concepts about thin-edge. In which scenarios thin-edge can be used. Thick Edge vs Thin edge, Graphical thin-edge architecture with all components.
+    1.1 data model - Explain the data model, graphically and explain each data bucket
+    1.2 runtime - What is the thin-edge runtime (components)
+    1.3 cloud transport - Explain the concepts & challenges of sending data to IoT Platform clouds and how this can be address with the cloud transport
+    1.4 mapper - Explain the concepts & challenges of mapping generic data to the thin-edge data model and cloud platforms and why this is necessary.
+    1.5 plugins - Explain the thin-edge plugin concept how to extend it with additional features. 
+    1.6 security - Explain how you can securely setup the thin-edge and communicate with cloud platforms using device certificates
```

Comment:
If this is supposed to be a TOC, the markup should be a list with links to the (right now not existing) files, like in [SUMMARY.md](https://raw.githubusercontent.com/thin-edge/thin-edge.io/main/docs/src/SUMMARY.md).

I guess that's also what @didier-wenzek meant above.